.github/workflows/doc_pdf_export.yaml

```diff
@@ -0,0 +1,148 @@
+name: PS2 Documentation Export 
+
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
Limit some branches would be better (release/develop/main)

Reply:
Updated to trigger only on the following branches:
- `'main'`
- `'**doc**'`
- `'release**'`

And the following tags:
- `"[0-9]+.[0-9]+.[0-9]+-rc[0-9]+"`
- `"[0-9]+.[0-9]+.[0-9]"`

commit id: https://github.com/COPRS/processing-sentinel-2/pull/11/commits/5e1902749a20ad2c352dd6a4ed67e2807afe47c3